catkit/hardware/iris_ao/segmented_dm_command.py

```diff
@@ -463,7 +455,7 @@ def plot_psf(self, rotation_angle, figure_name_prefix, out_dir, vmin=10e-8,
             plt.show()
 
 
-def load_command(segment_values, dm_config_id, wavelength, testbed_config_id,
+def load_command(segment_values, dm_config_id, display_wavelength,
```

Comment:
(Disclaimer: I recognize that you aren't necessarily the author of the parts that I am commenting on but since this was updated it drew my attention to it) 

I am pretty uncomfortable with needing to include a wavelength when loading a command. I recommend removing it as an input parameter for `SegmentedDmCommand`, and only require it as a parameter for the `display` method.

Reply:
> I am pretty uncomfortable with needing to include a wavelength when loading a command. I recommend removing it as an input parameter for SegmentedDmCommand, and only require it as a parameter for the display method.

Great - I'm all for that change too! I wasn't sure how much I should minimize changes to avoid complications for JOST, but I definitely agree it's better to remove any of the display stuff from when needing to load a command. I'll add that in here too. Thanks.